src/__tests__/NavBar.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import {render} from "@testing-library/react"
+import NavBar from "../components/NavBar"
+import { BrowserRouter } from "react-router-dom";
+
+describe("NavBar", ()=>{
+    it("renders correctly", ()=>{
+        const {asFragment} = render(<BrowserRouter><NavBar /></BrowserRouter>)
+        expect(asFragment).toMatchSnapshot()
+    })
+})
```

Comment:
Although snapshot tests are useful, you shouldn't solely rely on them for tests a component. Try to think what you can test for. For this NabBar, I would have a test to check that the logo is rendered with the correct alt tag and that the NavLinks are present, with the correct `href` (when `NavLink` is rendered to the DOM it become and `a` element with the `href` being what you set the `to` to)